### PR TITLE
teams: show member details (fixes #3546)

### LINKED ISSUE #3546: view member details in teams/enterprise
`planet`
https://github.com/open-learning-exchange/myplanet/assets/64019708/3b62d365-8844-480b-a219-5e686b269493

on click of the member items we should be able to view other member details currently on `myPlanet` nothing happens on click

https://github.com/open-learning-exchange/myplanet/assets/64019708/5d18b2c9-d626-48aa-b228-c6271de2591a

### PR DESCRIPTION
**Pull request by rlam20**
fixes #3546 
Matches info stored in planet 
image

### CODE REVIEW

**Review by ewhang5 — CHANGES_REQUESTED** [review]
seems like the profile pages show a white box rather than the profile picture that they have set.

**Comment by strawberrybread** [comment]
is there a way for users to hide their personal info like DOB, email, phone number?

**Comment by Okuro3499** [comment]
@strawberrybread no they can't hide

File: app/src/main/java/org/ole/planet/myplanet/ui/team/teamMember/AdapterJoinedMember.kt [hunk 1]
```diff
@@ -19,39 +20,61 @@ import org.ole.planet.myplanet.utilities.Utilities
...
 
     override fun onBindViewHolder(holder: ViewHolderUser, position: Int) {
-        rowJoinedUserBinding.tvTitle.text = if (list[position].toString() == " ") list[position].name else list[position].toString()
-        rowJoinedUserBinding.tvDescription.text = "${list[position].getRoleAsString()} (${RealmTeamLog.getVisitCount(mRealm, list[position].name, teamId)} ${context.getString(R.string.visits)})"
+        val user = list[position]
+        rowJoinedUserBinding.tvTitle.text = user.name ?: user.toString()
+        rowJoinedUserBinding.tvDescription.text = "${user.getRoleAsString()} (${RealmTeamLog.getVisitCount(mRealm, user.name, teamId)} ${context.getString(R.string.visits)})"
 
-        Glide.with(context)
-            .load(list[position].userImage)
```
Comment (Okuro3499) [inline app/src/main/java/org/ole/planet/myplanet/ui/team/teamMember/AdapterJoinedMember.kt:37]: this is the line that show the image. why is it removed? is it replaced by something else?

**Review by Okuro3499 — CHANGES_REQUESTED** [review]
@rlam20 works great and would be better if you show the profile images

**Comment by Okuro3499** [comment]
Screenshot_20240614_223350

**Review by Okuro3499 — APPROVED** [review]

**Comment by ewhang5** [comment]
i think this could look even better if the profile pictures are in the shape of a circle, but maybe that is for a future issue. looks great right now too

**Review by ewhang5 — APPROVED** [review]

**Review by strawberrybread — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**